Src/AutoFixtureUnitTest/Kernel/SeededFactoryTest.cs

```diff
@@ -129,5 +129,23 @@ public void CreateWithCorrectlyTypedSeededRequestWithNullSeedReturnsCorrectResul
             Assert.Equal(expectedResult, result);
             // Teardown
         }
+
+        [Fact]
+        public void CreateWithCorrectlyTypedRequestReturnsCorrectResult()

```

Comment:
Yes, that test was superseded by this [scenario test](https://github.com/AutoFixture/AutoFixture/blob/467-create-value-from-seed/Src/AutoFixtureUnitTest/FixtureTest.cs#L3525;L3534) introduced in 28bb833. I'll remove it.